soroban-spec/src/gen/ts/project_template/package.json

```diff
@@ -0,0 +1,18 @@
+{
+    "version": "0.0.0",
+    "name": "INSERT_CONTRACT_NAME_HERE",
+    "dependencies": {
+        "@stellar/freighter-api": "1.4.0",
+        "buffer": "6.0.3",
+        "soroban-client": "0.7.2"
+    },
+    "scripts": {
+        "build": "tsc"
+    },
+    "type": "module",
+    "main": "out/index.js",
```

Comment:
Oh, just noticed. I feel like the more usual js name for this dir is `dist` instead of `out`.